src/Csv/CsvFile.fs

```diff
@@ -91,25 +92,22 @@ and CsvFile private (readerFunc:Func<TextReader>, [<Optional>] ?separators, [<Op
     new CsvFile(readerFunc, ?separators=separators, ?quote=quote, ?hasHeaders=hasHeaders, ?ignoreErrors=ignoreErrors, ?skipRows=skipRows)
 
   /// Loads CSV from the specified uri
-  static member Load(uri:string, [<Optional>] ?separators, [<Optional>] ?quote, [<Optional>] ?hasHeaders, [<Optional>] ?ignoreErrors, [<Optional>] ?skipRows) = 
-    let separators = defaultArg separators ""    
-    let separators = 
-        if String.IsNullOrEmpty separators && uri.EndsWith(".tsv" , StringComparison.OrdinalIgnoreCase) 
-        then "\t" else separators
-    let readerFunc = Func<_>(fun () -> asyncReadTextAtRuntime false "" "" "CSV" "" uri |> Async.RunSynchronously)
-    new CsvFile(readerFunc, separators, ?quote=quote, ?hasHeaders=hasHeaders, ?ignoreErrors=ignoreErrors, ?skipRows=skipRows)
+  static member Load(uri:string, [<Optional>] ?separators, [<Optional>] ?quote, [<Optional>] ?hasHeaders, [<Optional>] ?ignoreErrors, [<Optional>] ?skipRows, [<Optional>] ?encoding) = 
+    CsvFile.AsyncLoad(uri, ?separators=separators, ?quote=quote, ?hasHeaders=hasHeaders, ?ignoreErrors=ignoreErrors, ?skipRows=skipRows) |> Async.RunSynchronously
 
   /// Loads CSV from the specified uri asynchronously
-  static member AsyncLoad(uri:string, [<Optional>] ?separators, [<Optional>] ?quote, [<Optional>] ?hasHeaders, [<Optional>] ?ignoreErrors, [<Optional>] ?skipRows) = async {
-    let separators = defaultArg separators ""    
+  /// This doesn't actually load the file async, so why is it here? The `Async.RunSynchronously` ruins everything.
```

Comment:
It only forces to download the entire file when you iterate .Rows to to get the first item, so in that sense it's still async. To be truly async you would have to use something like AsyncSeq which would complicate the usage quite a bit for the common case. I'd be welcome to have one variation that uses it though, but it's actually quite complicated to have both (I tried long time ago), and then you add a dependency to another library.
Can you remove the comment in the next iteration? It's using triple slash so it's going to the docs